tests/basic_evm_tester.hpp

```diff
@@ -54,13 +54,22 @@ void to_variant(const evmc::address& o, fc::variant& v);
 
 namespace evm_test {
 
-struct evmtx_v0 {
+struct evmtx_base {
    uint64_t  eos_evm_version;
    bytes     rlptx;
-   uint64_t  base_fee_per_gas;
 };
 
-using evmtx_type = std::variant<evmtx_v0>;
+struct evmtx_v1 : evmtx_base {
```

Comment:
Shall we still keep the base price in evmtx_v3 so that it would be possible to implement the option to still use old model with high eos evm version?